spec/models/rent_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe Rent do
+  it 'is valid with valid attributes' do
+    rent = build(:rent)
+
+    expect(rent).to be_valid
+  end
+
+  it 'is not valid without a user' do
+    rent = build(:rent, user: nil)
+
+    expect(rent).not_to be_valid
+  end
+
+  it 'is not valid without a book' do
+    rent = build(:rent, book: nil)
+
+    expect(rent).not_to be_valid
+  end
+
+  it 'is not valid without a start_date' do
+    rent = build(:rent, start_date: nil)
+
+    expect(rent).not_to be_valid
+  end
+
+  it 'is not valid without an end_date' do
+    rent = build(:rent, end_date: nil)
+
+    expect(rent).not_to be_valid
+  end
+
+  it 'is not valid when end_date < start_date' do
+    rent = build(:rent, start_date: Time.current, end_date: 2.days.ago)
+    rent.valid?
```

Comment:
Unnecessary line

Reply:
@facucompe I tried using create instead of build, but the test fails before getting to the expectations. Using .valid? was the only way I found to trigger the validations while also being able to test my custom error message (build does not trigger validations). Could you please point me in the right direction?

Reply:
I think you should be able to do this:
```
subject { build(:rent, start_date: Time.current, end_date: 2.days.ago) }
it { is_expected.not_to be_valid }
```